### PR TITLE
Add copy button and serial number to ItemsTable 

### PR DESCRIPTION
**Pull request by hobbitronics**
- add CopyTableButton
- add serial number and button to show/hide it on ItemsTable
copytable
Added make and model
image
They are hidden by default
image

### CODE REVIEW

**Comment by Schparky** [comment]
What would it look like to change the "show/hide serial #" to "show all columns" and "hide make/model/serial number columns". I don't really know what data people want, but it might help to look forward a bit.

**Comment by hobbitronics** [comment]
> What would it look like to change the "show/hide serial #" to "show all columns" and "hide make/model/serial number columns". I don't really know what data people want, but it might help to look forward a bit.

That would be pretty easy. We can do all together or separate them out individually and maybe use checkboxes instead of the button.

File: src/components/Datatable/ItemsTable.svelte [hunk 1]
```diff
@@ -65,6 +87,7 @@ const columns: Column[] = [
     sortable: true,
   },
 ]
+const itemsTableId = `items-table-${random(1000000, 9999999).toString()}`
```
Comment (Schparky) [inline src/components/Datatable/ItemsTable.svelte:90]: would `generateRandomID` work here?

**Review by Schparky — APPROVED** [review]

**Comment by Schparky** [comment]
nice!

**Pull request merged**